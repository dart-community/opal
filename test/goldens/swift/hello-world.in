let message: String = "Hello, world!"

// Print the message.
print("The message is: \(message)")
