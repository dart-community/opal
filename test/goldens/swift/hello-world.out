let message: String = "Hello, world!"
^^^
source-code-swift-keyword-declaration-let
   ^
   source-code-swift-whitespace
    ^^^^^^^
    source-code-swift-identifier
           ^
           source-code-swift-keyword-operator
            ^
            source-code-swift-whitespace
             ^^^^^^
             source-code-swift-identifier-type-built-in
                   ^
                   source-code-swift-whitespace
                    ^
                    source-code-swift-keyword-operator
                     ^
                     source-code-swift-whitespace
                      ^
                      source-code-swift-literal-string-quoted-double-literal-string-quoted-double-begin
                       ^^^^^^^^^^^^^
                       source-code-swift-literal-string-quoted-double-literal-string-unquoted-literal-string-unquoted
                                    ^
                                    source-code-swift-literal-string-quoted-double-literal-string-quoted-double-end

// Print the message.
^^^^^^^^^^^^^^^^^^^^^
source-code-swift-comment-line
print("The message is: \(message)")
^^^^^
source-code-swift-identifier-function
     ^
     source-code-swift-punctuation
      ^
      source-code-swift-literal-string-quoted-double-literal-string-quoted-double-begin
       ^^^^^^^^^^^^^^^^
       source-code-swift-literal-string-quoted-double-literal-string-unquoted-literal-string-unquoted
                       ^^^^^^^^^^
                       source-code-swift-literal-string-quoted-double-literal-string-unquoted-literal-string-interpolation
                                 ^
                                 source-code-swift-literal-string-quoted-double-literal-string-quoted-double-end
                                  ^
                                  source-code-swift-punctuation
